third_party/conan/docker/Dockerfile.clang11_opengl_qt

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2021 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+FROM ubuntu:20.04
+
+ADD llvm-snapshot.gpg.key /root/
+
+RUN useradd -Ums /bin/bash conan \
+    && apt-get -qq update \
+    && DEBIAN_FRONTEND=noninteractive \
+        apt-get install -y --no-install-recommends \
```

Comment:
Optional super-nit: Internally, we always use the "full" parameter version if possible, i.e. "--yes" for readability.